lib/utils.js

```diff
@@ -40,11 +40,20 @@ function wrapWithBackslash(str, options) {
     return out;
 }
 
+function removeTags(str){
+    if ( str == null){
```

Comment:
We should always use `===` to compare with `null`, to avoid accidental matches (`undefined` is `== null`, for example)